portfolio.js

```diff
@@ -0,0 +1,18 @@
+// const togglerButton = document.querySelector('#menuIcon');
+// const displayElement = document.querySelector('.desktop-list');
+// const exitIcon = document.querySelector('#exit-icon');
+// const body = document.querySelector('body')
+
+// togglerButton.addEventListener('click', () => {
+//   displayElement.style.display = 'block';
+//   togglerButton.style.display = 'none'
+//   console.log('#exitIcon')
+//   body.classList.toggle('open');
+
+// });
+const toggle = document.querySelector('.toggle');
+const body = document.querySelector('body');
+
+toggle.addEventListener('click', () => {
+  body.classList.toggle('open');
+});
```

Comment:
- [ ] I noticed that when I click one of the links in the mobile menu, it redirects me to the part of the page but the mobile menu doesn't close. I would suggest that you fix this issue as it is a requirement for the mobile menu to close when a link in the mobile menu is clicked. 👍 😎 

![chrome-capture-2023-2-14](https://user-images.githubusercontent.com/77106962/225132481-c863a2b2-460c-45ce-872c-8ca4709a06cb.gif)